src/features/defi/components/Overview/Overview.tsx

```diff
@@ -226,6 +234,17 @@ export const Overview: React.FC<OverviewProps> = ({
                 </StatGroup>
               )}
               {postChildren}
+              {provider === DefiProvider.ThorchainSavers ? (
```

Comment:
preferably-blocking: can we use at the very least some `isThorfiHalted` constant/feature flag of sorts somewhere, and ideally, use mimirs from THORNode?

Not sure which one we can use in case we want to go with the latter, I *believe* `PAUSELOANS` would work here, as savers are effectively disabled as a result of the above.

Reply:
Changed in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)